apache2/apache2_config.c

```diff
@@ -30,6 +30,13 @@
     APLOG_USE_MODULE(security2);
 #endif
 
+# Returns the rule id if existing, otherwise the file name & line number
```

Comment:
This is not a `C` comment, it's a preprocessor directive. You should replace it by this:
```C
// Returns the rule id if existing, otherwise the file name & line number
```

Reply:
Done. I wanted to be too smart...